cpp/daal/src/algorithms/dtrees/gbt/gbt_predict_dense_default_impl.i

```diff
@@ -45,46 +45,68 @@ typedef float ModelFPType;
 typedef uint32_t FeatureIndexType;
 const FeatureIndexType VECTOR_BLOCK_SIZE = 64;
 
-template <typename algorithmFPType, typename DecisionTreeType, CpuType cpu>
-inline void predictForTreeVector(const DecisionTreeType & t, const FeatureTypes & featTypes, const algorithmFPType * x, algorithmFPType v[])
+template <bool hasUnorderedFeatures, bool hasAnyMissing>
+struct PredictDispetcher {
+    typedef PredictDispetcher<hasUnorderedFeatures, hasAnyMissing> type;
+};
+
+template<typename algorithmFPType>
+inline FeatureIndexType UpdateIndex(FeatureIndexType idx, algorithmFPType valueFromDataSet, const ModelFPType * splitPoints, const int * yesIfMissing,
```

Comment:
Function names should start with small letters, i.e. `updateIndex()`

Reply:
done